mlir/tools/mlir-rocm-runner/mlir-rocm-runner.cpp

```diff
@@ -61,6 +76,25 @@ static cl::opt<bool>
                cl::desc("input is in the MLIR LLVM/ROCDL dialect"),
                cl::init(false));
 
+static constexpr const char kTargetTriple[] = "amdgcn-amd-amdhsa";
+
+#if __INCLUDE_HIP__
+namespace {
+    void getGpuGCNArchName(hipDevice_t device, std::string &gcnArchName) {
```

Comment:
`static` over anonymous namespaces for things like this, per upstream style we haven't always known about or followed well

And you'll want to `git clang-format` after staging and before committing (or `git clang-format miopen-dialect` after committing) or else your changes will get bounced back by CI for not being formatted correctly.

Reply:
Thank you @krzysz00 for the advice. I'll make sure I run it before submitting commits.

As per the discussion with @jerryyin, the next step will be to remove calling hipGetDeviceProperties() inside mlir-rocm-runner to avoid the out-of-handle problem when running multiple rocm instances concurrently. 
I'll check with the coding style to learn more about things like this.